common/src/main/cpp/extensions/openxr_fb_passthrough_extension_wrapper.cpp

```diff
@@ -607,6 +619,27 @@ bool OpenXRFbPassthroughExtensionWrapper::has_layer_depth_passthrough_capability
 	return (system_passthrough_properties.capabilities & XR_PASSTHROUGH_CAPABILITY_BIT_FB) && (system_passthrough_properties.capabilities & XR_PASSTHROUGH_CAPABILITY_LAYER_DEPTH_BIT_FB);
 }
 
+bool OpenXRFbPassthroughExtensionWrapper::is_passthrough_preferred() {
+	if (!meta_passthrough_preferences_ext) {
+		UtilityFunctions::print("Meta passthrough preferences extension is not enabled");
+		return false;
+	}
+
+	XrPassthroughPreferencesMETA passthrough_preferences = {
+		XR_TYPE_PASSTHROUGH_PREFERENCES_META, // type
+		nullptr, // next
+		0, // flags
+	};
+
+	XrResult result = xrGetPassthroughPreferencesMETA(SESSION, &passthrough_preferences);
+	if (XR_FAILED(result)) {
+		UtilityFunctions::print("Failed to get passthrough preferences, error code: ", result);
```

Comment:
Can you update this `print` to `printerr` as discussed in the other PR.

Reply:
Updated on latest push!